swap_meet/vendor.py

```diff
@@ -1,2 +1,125 @@
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self, inventory=None):
+        if not inventory:
+            inventory = []
+        self.inventory = inventory
+    
+    def add(self, item):
+        """Adds an item to the instance's inventory and returns it."""
+        self.inventory.append(item)
+        return item
+    
+    def remove(self, item):
+        """
+        Removes items from the instance's inventory.
+
+        Parameter: 
+        item (str): Inventory item to be removed.
+
+        Returns:
+        bool: True if item is inventory, False otherwise.
+        """
+
+        if item in self.inventory:
+            self.inventory.remove(item)
+            return item
+        return False
```

Comment:
I like inverting the logic of the condition to be able to treat it like a guard clause, which allows us to outdent the main logic of the function something like the following:
```python
        if item not in self.inventory:
            return False

        self.inventory.remove(item)
        return item
```

Another approach we could take is to try to remove the item directly, and handle the `ValueError` that occurs if it's not there, and return `False` to handle it.